app/assets/javascripts/application.js

```diff
@@ -70,41 +71,45 @@ $(document).ready(function() {
 
   // For DataTables and Bootstrap
   $('.datatable').dataTable({
-    "sDom": "<'row'<'span4'l><'span5'f>r>t<'row'<'span3'i><'span6'p>>",
-    "sPaginationType": "bootstrap",
-    "sScrollX": "100%",
-    "aoColumnDefs": [
-          { "bSortable": false, "aTargets": [ "no_sort" ] }
-        ]
+    "pagingType": "full_numbers",
+    "scrollX": false,
+    "columnDefs": [
+      { "orderable": false, "targets": [ "no_sort" ] }
+    ]
   });
 
-  wideDataTables = $('.datatable-wide').dataTable({
-    "sDom": "<'row'<'span5'l><'span7'f>r>t<'row'<'span5'i><'span7'p>>",
-    "sPaginationType": "bootstrap",
-    "sScrollX": "100%",
-    "aoColumnDefs": [
-          { "bSortable": false, "aTargets": [ "no_sort" ] }
-        ]
+  var wideDataTables = $('.datatable-wide').dataTable({
```

Comment:
It's not used anywhere else, as far as I can tell, so let's get rid of the assignment.